doc/user/environments.xml

```diff
@@ -1876,9 +1876,9 @@ env = Environment()
 env.Command('foo', [], '__ROOT__/usr/bin/printenv.py')
       </file>
       <file name="__ROOT__/usr/bin/printenv.py" chmod="0o755">
-#!/usr/bin/env python
```

Comment:
Why remove this line?

Reply:
hmmm, wasn't intentional (but it's also inside an xml comment so ignored). will restore.